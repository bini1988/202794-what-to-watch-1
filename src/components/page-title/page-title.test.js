import React from "react";
import renderer from "react-test-renderer";
import PageTitle from "./page-title.jsx";

it(`PageTitle correctly renders default markup`, () => {
  const tree = renderer.create(
      <PageTitle
        className="title-class">
        {`Page Title`}
      </PageTitle>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it(`PageTitle correctly renders hidden mode markup`, () => {
  const tree = renderer.create(
      <PageTitle
        className="title-class"
        hidden={true}>
        {`Page Title`}
      </PageTitle>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
