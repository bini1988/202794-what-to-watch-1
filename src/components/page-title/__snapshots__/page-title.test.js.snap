// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle correctly renders default markup 1`] = `
<h1
  className="page-title title-class"
>
  Page Title
</h1>
`;

exports[`PageTitle correctly renders hidden mode markup 1`] = `
<h1
  className="page-title visually-hidden title-class"
>
  Page Title
</h1>
`;
